strongbox-web-core/src/main/java/org/carlspring/strongbox/converters/PrivilegeListFormToPrivilegeListConverter.java

```diff
@@ -0,0 +1,34 @@
+package org.carlspring.strongbox.converters;
+
+import org.carlspring.strongbox.forms.PrivilegeForm;
+import org.carlspring.strongbox.forms.PrivilegeListForm;
+import org.carlspring.strongbox.security.Privilege;
+import org.carlspring.strongbox.security.Role;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.core.convert.ConversionService;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Pablo Tirado
+ */
+public class PrivilegeListFormToPrivilegeListConverter
+        implements Converter<PrivilegeListForm, List<Privilege>>
+{
+
+    @Override
+    public List<Privilege> convert(PrivilegeListForm privilegeListForm)
+    {
+        List<Privilege> privilegeList = new ArrayList<>();
+        for (PrivilegeForm privilegeForm : privilegeListForm.getPrivileges())
+        {
+            Privilege privilege = new Privilege(privilegeForm.getName(), privilegeForm.getDescription());
+            privilegeList.add(privilege);
+        }
+
+        return privilegeList;
+    }
+}
```

Comment:
Some of your classes (here and below) don't have an empty line at the end of the file. Could you please fix this? I believe there's a setting in each IDE to add new lines to the end of files.

Reply:
@ptirador : Thanks for fixing this! :)